src/cmdstan/arguments/argument_parser.hpp

```diff
@@ -19,9 +19,12 @@ class argument_parser {
 
   int parse_args(int argc, const char *argv[], stan::callbacks::writer &info,
                  stan::callbacks::writer &err) {
+    if (argc == 0) {
+      return stan::services::error_codes::USAGE;
```

Comment:
This should basically never happen right? Like `argc >= 1` cause the first argv will be the name of the program executing things?

Reply:
yup, should never happen - but there's  unit test for it - get rid of both?

Reply:
removed those lines - not necessary.
leaving unit test alone.